tests/api/flights.test.ts

```diff
@@ -1,7 +1,25 @@
 import { flights } from '../../src/api/flights';
 import { createTestApp, testHandler } from '../testUtils/testHandler';
 
+jest.mock('../../src/services/generateFlightsBySeed.ts', () => ({
+  generateFlightsBySeed: jest.fn().mockReturnValue([
+    {
+      flightNumber: '4858',
+      origin: { code: 'DFW', city: 'Dallas-Fort Worth', timezone: 'America/Chicago', location: { latitude: 32.8998, longitude: 97.0403 } },
+      destination: { code: 'PHL', city: 'Philadelphia', timezone: 'America/New_York', location: { latitude: 39.8729, longitude: -75.2437 } },
+      distance: 7393,
+      duration: { locale: '19h 3m', hours: 19, minutes: 3 },
+      departureTime: '2020-01-01T03:46:40.000-05:00',
+      arrivalTime: '2020-01-01T23:49:40.000-04:00',
+      aircraft: { model: '738', passengerCapacity: { total: 160, main: 144, first: 16 }, speed: 400 },
+    },
+  ]),
+}));
+
 describe('flights', () => {
+  beforeEach(() => {
+    jest.resetModules();
+  });
```

Comment:
I think this was supposed to be `resetMocks`
```suggestion
  beforeEach(() => {
    jest.resetMocks();
  });
  
```